terraform/layer1-aws/aws-eks.tf

```diff
@@ -72,60 +63,10 @@ module "eks" {
 
   vpc_id = module.vpc.vpc_id
 
-  cluster_addons = local.eks_addons
-
-  cluster_encryption_config = var.eks_cluster_encryption_config_enable ? [
-    {
-      provider_key_arn = aws_kms_key.eks[0].arn
-      resources        = ["secrets"]
-    }
-  ] : []
-
   cluster_endpoint_public_access       = var.eks_cluster_endpoint_public_access
   cluster_endpoint_private_access      = var.eks_cluster_endpoint_private_access
   cluster_endpoint_public_access_cidrs = var.eks_cluster_endpoint_only_pritunl ? ["${module.pritunl[0].pritunl_endpoint}/32"] : ["0.0.0.0/0"]
 
-  # Extend cluster security group rules
-  cluster_security_group_additional_rules = {
```

Comment:
Why have these security groups been deleted?

Reply:
in the eks module, they created groups and turned them on by default